cypress/fixtures/examples/good/ds003653_participant_annotated_1698934398962.json

```diff
@@ -1,123 +0,0 @@
-{
```

Comment:
Why delete this file?

Reply:
good point. Brought it back. And also made an issue to clean it up properly in #838